app/models/stats/stats_report.rb

```diff
@@ -28,9 +28,7 @@ class StatsReport < ApplicationRecord
     scope :management_information, -> { not_errored.where(report_name: 'management_information') }
     scope :provisional_assessment, -> { not_errored.where(report_name: 'provisional_assessment') }
 
-    has_attached_file :document, s3_headers.merge(REPORTS_STORAGE_OPTIONS)
-
-    validates_attachment_content_type :document, content_type: ['text/csv']
```

Comment:
not so important for reports but for messages and [evidence] documents we would need to have a validation on the content-type.  Is that possible? and if so I'd say it is worth doing it for reports too, if for no other reason than to prove we can later for documents etcs

Reply:
I removed the validation because the file is being created in the app. Potentially validation could fail if something goes wrong with the CSV creation, indicating a bug. I can put it back in but I'm not sure of the value.

There is (and will remain) validation for uploads and there are tests for this.